doc/source/conf.py

```diff
@@ -74,7 +74,7 @@
 #
 # This is also used if you do content translation via gettext catalogs.
 # Usually you set "language" from the command line for these cases.
-language = None
+language = 'en'
```

Comment:
Is this needed with more recent versions of sphinx? This change is not in master. Is there a reason for this difference?

Reply:
@juergbi Yeah, it will fail with a warning saying the language "None" is not supported

warnings are activated as errors and that's why the build will fail: maybe that is not activated in master

Reply:
Can you create issue about this for master? It sounds like a bug that the warning is simply ignored.

Reply:
Sure, I will